src/lsdb/core/crossmatch/kdtree_match.py

```diff
@@ -82,3 +87,8 @@ def _get_point_coordinates(self) -> Tuple[npt.NDArray[np.float64], npt.NDArray[n
             lat=self.right[self.right_catalog_info.dec_column].to_numpy(),
         )
         return left_xyz, right_xyz
+
+    def _get_extra_columns_df(self, chord_distances: npt.NDArray[np.float64]) -> pd.DataFrame:
```

Comment:
Would it be easier to append meta if this is constructed from the start as a NestedFrame?